addons/l10n_lt/data/account.account.template.csv

```diff
@@ -0,0 +1,184 @@
+"account_account_template_2442","Prepaid Profit Tax",2442,"account.data_account_type_receivable","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_2_4","True"
+"account_account_template_2443","Tax Overpayments",2443,"account.data_account_type_receivable","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_2_4","True"
+"account_account_template_2444","VSDF (Lithuanian State Social Insurance Fund) Debt to the Company",2444,"account.data_account_type_receivable","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_2_4","True"
+"account_account_template_24450","Value of Receivables from Accountable Persons",24450,"account.data_account_type_receivable","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_2_4","True"
+"account_account_template_24460","Value of Other Receivable Debts",24460,"account.data_account_type_receivable","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_2_4","True"
+"account_account_template_24471","Profit paid in Advance to the Owners of the Company",24471,"account.data_account_type_receivable","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_2_4","True"
+"account_account_template_24472","Payed Funds for Personal Use to the Owners of the Company",24472,"account.data_account_type_receivable","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_2_4","True"
+"account_account_template_2449","Other Questionable Debts (-)",2449,"account.data_account_type_receivable","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_2_4","True"
+"account_account_template_26200","Acquisition Cost of Equity Securities of Other Companies",26200,"account.data_account_type_current_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_3_2","False"
+"account_account_template_26209","Value Loss of Equity Securities of Other Companies (-)",26209,"account.data_account_type_current_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_3_2","False"
+"account_account_template_262110","Acquisition Cost of Other Non-Equity Securities",262110,"account.data_account_type_current_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_3_2","False"
+"account_account_template_262119","Value Loss of Other Non-Equity Securities (-)",262119,"account.data_account_type_current_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_3_2","False"
+"account_account_template_2623","Term Deposits",2623,"account.data_account_type_current_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_3_2","False"
+"account_account_template_273","Cash in Transit",273,"account.data_account_type_current_assets","account_chart_template_lithuania","l10n_lt.account_account_tag_b_4","True"
```

Comment:
AFAIK, all lithuanian companies just use this one account `273` for cash transfers. With current configuration, there is also `2731` being generated when loading CoA. It gets created as `Liquidity Transfer`.
![image](https://user-images.githubusercontent.com/17097810/65612893-c0532c00-dfbd-11e9-9965-53adaea12415.png)

We have our own localization module and we have found a way to "force" Odoo to just recognize the included account, instead.
It is done by simply making the id look like this:
```suggestion
"account_chart_template_lithuania_liquidity_transfer","Cash in Transit",273,"account.data_account_type_current_assets","account_chart_template_lithuania","l10n_lt.account_account_tag_b_4","True"
```

Odoo code that makes this happen:
https://github.com/odoo/odoo/blob/25b8aeb9e2e9f2d1a3745ce5e7ee43ed3eef6872/addons/account/models/chart_template.py#L614-L617

Now I know that this looks kind of "hack'y", but it really works well and we haven't seen any problems with it. Also, it looks like this part of code is there to stay for v13 as well. Would really like to hear your, @jco-odoo , opinion on this.

Reply:
Well I guess we can do that, if its intended way to do it?

Reply:
We need someone from Odoo to answer this.
As far as I've seen, it works. If the code doesn't find an account by this id, it then creates it. This is how it looks exported without the change:
```csv
"id","name","code","user_type_id/id"
"l10n_lt.1_account_account_template_273","Cash in Transit","273","account.data_account_type_current_assets"
"l10n_lt.1_account_chart_template_lithuania_liquidity_transfer","Liquidity Transfer","2731","account.data_account_type_current_assets"
```

If the account with such id is there, it simply uses it instead.